bot/handlers/administration/add_whitelist_handler.py

```diff
@@ -24,33 +23,39 @@ def get_commands(self) -> List[str]:
     def _get_validator_functions(self) -> ValidatorFunctions:
         return [
             self.__check_argument_count,
+            self.__check_user_id_is_digit,
         ]
 
     async def __check_argument_count(self, message: Message) -> bool:
         return await self._validate_argument_count(
             message, 2, get_no_username_provided_message(),
         )
 
+    async def __check_user_id_is_digit(self, message: Message) -> bool:
+        user_input = message.text.split()[1]
+        if not user_input.isdigit():
+            await self.__reply_user_not_found(message)
+            return False
+        return True
+
     async def _do_handle(self, message: Message) -> None:
         user_input = message.text.split()[1]
-        try:
-            user_data = await self._bot.get_chat(int(user_input) if user_input.isdigit() else user_input)
-        except TelegramBadRequest as e:
-            if "chat not found" in str(e).lower():
-                await self._answer(message, get_user_not_found_message())
-                return
-            raise
 
         await DatabaseManager.add_user(
-            user_id=user_data.id,
-            username=user_data.username,
-            full_name=user_data.full_name,
+            user_id=int(user_input),
+            username="",
+            full_name="",
             note=None,
         )
```

Comment:
A nie byłoby lepiej zbierać te wszystkie dane z /addwhitelist? bo trochę nie czaję po co to rozbijać tak że potem musisz ręcznie edytować

Reply:
Bo nie da się. W sensie żebym ja mógł pobrać nazwę usera nick itp na podstawie id to ktoś musiał już mieć jakiś kontakt z botem. A komenda powstała po to żeby móc jakoś manualnie coś na szybko wrzucić do bazy bo wchodzenia na postgresa.

Reply:
Bo nie da się. W sensie żebym ja mógł pobrać nazwę usera nick itp na podstawie id to ktoś musiał już mieć jakiś kontakt z botem. A komenda powstała po to żeby móc jakoś manualnie coś na szybko wrzucić do bazy bo wchodzenia na postgresa.

Reply:
Więc to co było jebało błędami jak dodawałem kogoś kto nie gadał z botem. A znowu podawanie w tej komendzie milion argumentów to też taki se deal ale można dodać jako opcjonalny parametr.